stubs/setuptools/METADATA.toml

```diff
@@ -1,4 +1,4 @@
-version = "68.2.*"
+version = "69.0.*"
 upstream_repository = "https://github.com/pypa/setuptools"
 partial_stub = true
```

Comment:
I'm pretty sure this was left there accidentally when `ignore_missing_stub = true` was removed in #10058 , probably because it happened so close to #10157
```suggestion
```

Reply:
Good catch!